Add OAS 3 information in API specification tasks with Design Center (#134)
mulesoft/docs-studio: modules/ROOT/pages/sync-imported-api-specifications-design-center.adoc

You can edit an imported API specification in your project as an API project.

> OAS 3.0 API specifications are not supported for importing as an API dependency for your project and therefore you cannot edit them following this process. To import an OAS 3.0 API specification as an editable API project see xref:sync-api-projects-design-center.adoc[synchronize API specifications with Design Center].

Reviewer: ```suggestion
Because importing OAS 3.0 API specifications as an API dependency for your project is not supported, you cannot edit them following this process. To import an OAS 3.0 API specification as an editable API project, see xref:sync-api-projects-design-center.adoc[Synchronize API Specifications with Design Center].
```
init. caps. if the xref is to a title; work into the sentence as plain lowercase narrative if not.
